dependencies/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_subdirectory(alfont)
-add_subdirectory(allegro)
+#add_subdirectory(allegro)
```

Comment:
@pbomta I can't decide if this is a good change or not, can you?

Reply:
Yes and/or no. 
On Linux system, i have installed allegro, so I didn't recompile it when i need to compile d2TM. Allegro is installed as another libraries. So I don't need it.